docs/sources/flow/reference/components/otelcol.connector.servicegraph.md

```diff
@@ -135,7 +135,7 @@ Name | Type | Description | Default | Required
 
 ### output block
 
-{{< docs/shared lookup="flow/reference/components/output-block-metrics.md" source="agent" >}}
+{{< docs/shared lookup="flow/reference/components/output-block-metrics.md" source="agent" version="<AGENT VERSION>" >}}
```

Comment:
I see a few other occasions of this in case we want to bundle them here.



```
$ grep -r "docs/shared lookup" | grep -v "VERSION"
docs/developer/writing-flow-component-documentation.md:{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
docs/developer/writing-flow-component-documentation.md:{{< docs/shared lookup="flow/stability/experimental.md" source="agent" >}}
docs/sources/flow/reference/components/discovery.consulagent.md:{{< docs/shared lookup="flow/reference/components/tls-config-block.md" source="agent" >}}
docs/sources/flow/reference/components/otelcol.connector.servicegraph.md:{{< docs/shared lookup="flow/stability/experimental.md" source="agent" >}}
docs/sources/flow/reference/components/otelcol.connector.servicegraph.md:{{< docs/shared lookup="flow/reference/components/output-block-metrics.md" source="agent" >}}
```

Reply:
I can add them into the PR...